pyshacl/constraints/core/shape_based_constraints.py

```diff
@@ -77,14 +76,13 @@ def evaluate(
     ):
         """
         Entrypoint for constraint evaluation.
-        :type executor: dataclass
+        :type executor: SHACLExecutor
```

Comment:
Thanks, I meant to change this in the previous release, but looks like I missed this one.

Reply:
No problem, I just noticed this when string-searching.